pkgs/os-specific/linux/cpuid2cpuflags/default.nix

```diff
@@ -0,0 +1,19 @@
+{ lib, stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "cpuid2cpuflags";
+  version = "12";
+
+  src = fetchurl {
+    url =
+      "https://github.com/projg2/${pname}/releases/download/v${version}/${pname}-${version}.tar.bz2";
+    sha256 = "sha256-nTf6BDFqNgFqoUuebbnbEgF7/F+gY93n5noY5NSHk9I=";
+  };
+
+  meta = src.meta // {
```

Comment:
Please do not use this pattern (even if it's neat) so we can quickly look at a glance the homepage, etc (statically).